site/docs/components/semantic-icon-provider/index.mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: Semantic icon provider
+data:
+  description: "`SemanticIconProvider` introduces a centralized approach to managing and customizing icons in Salt’s ecosystem. This provider allows for flexible configuration of icon components and ensures consistent icon usage across your application."
+
+  # Fill in the info from the content template's "Metadata" table below.
+  # To omit optional items, comment them out with #
+  sourceCodeUrl: "https://github.com/jpmorganchase/salt-ds/tree/main/packages/core/src/salt-provider"
+  package:
+    name: "@salt-ds/core"
+  alsoKnownAs: ["Provider"]
+  relatedComponents: []
+
```

Comment:
If possible, add the available from version

Reply:
added key, will add later on once we decide to release